confgenerator/logging_processors.go

```diff
@@ -23,6 +25,15 @@ func (r LoggingProcessorParseJson) Type() string {
 	return "parse_json"
 }
 
+func (p LoggingProcessorParseJson) Components(tag string, i int) []fluentbit.Component {
```

Comment:
[Optional] Might be worth renaming the parameter of `Type()` above to `p` as well… Ditto for `LoggingProcessorParseRegex`.

Reply:
I am trying to minimize the diff. I agree it's silly the other one is r, but... cleanup can be separate.